### ISSUE TITLE
Glob stays in the tree's root path

### ISSUE THREAD

**Issue by AnatoleLucet** [benawad/destiny#88]
![](https://i.gyazo.com/49d5f12cecb082b0544692d39a7d879c.png)

The root file of the tree is `src/**/*.*`, it should probably be a simple `src`.

**Comment by sQVe** [comment]
What would you expect in cases like "**" or any other more open glob then?

The cli shows that the user inputted and what the result of that input was.

**Comment by AnatoleLucet** [comment]
Yes you're right, I haven't thought about that. We will keep it that way then.

**Comment by benawad** [comment]
The alternative would be to show the shared parent

**Comment by sQVe** [comment]
@benawad That is somewhat confusing when folders are ignored. It would look like we want to remove them.

**Comment by sQVe** [comment]
I'd say we have two alternatives. Either we keep it like it currently works or when a glob is inputted we get all root paths of that glob. That would probably be pretty clear to the user.

**Comment by AnatoleLucet** [comment]
> That is somewhat confusing when folders are ignored. It would look like we want to remove them.

What about somthing like this:
```
src
├──index
│  ├──formatFileStructure
│  │  ├──fixImports
│  │  │  └──makeImportPath.ts
│  │  ├──fixImports.ts
│  │  ├──moveFiles.ts
│  │  └──removeEmptyFolders.ts
│  ├──generateTrees (ignored)
├──shared
│  └──logger.ts
└──index.ts
```
`generateTrees (ignored)` in a gray color?

**Comment by benawad** [comment]
I like it

**Comment by SimonJang** [comment]
I fixed the log with the glob pattern. How do you see the implementation of the ignored files? I need some more information before this could actually be implemented.

- Is it graying out the ignored files?
- Is it graying out a folder that contains only ignored files?
- Is it graying out unused files?

Perhaps we need to make another issue since it looks more like a feature then a bug?

**Comment by AnatoleLucet** [comment]
@SimonJang It would be nice if all the cases you described are logged as ignored files/folders.
If a folder is ingored or all his files are, it should just log the folder name in gray with the `(ignored)` suffix. It shouldn't log his child(s) only the folder name.

We will keep this issue for now, I've changed the labels.
